tasty_delivery/core/domain/entities/order.py

```diff
@@ -0,0 +1,39 @@
+from pydantic import Field
+
+from core.domain.entities.base_entity import Base
+from core.domain.entities.client import Client
+
+
+class OrderIN(Base):
+    client: Client = Field()
+    preco: float = Field(gt=0)
+    lanche: str | None = Field(None)
+    acompanhamento: str | None = Field(None)
+    bebida: str | None = Field(None)
```

Comment:
Acredito que podemos remover esses caras e abstrair somente em um atributo chamados de "produtos", ou seja, os produtos do pedido. Aqui, podemos receber apenas o uuid dos produtos inclusos no pedido. Algo assim:
```suggestion
    produtos: Annotated[List, UUID] = Field(None)
```

Lembrando que é tudo sugestão, tá?

Reply:
faz sentido!